app/controllers/decidim/questions/admin/questions_moves_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Questions
+    module Admin
+      class QuestionsMovesController < Admin::ApplicationController
+        def create
+          enforce_permission_to :merge, :questions
+
+          @form = form(Admin::QuestionsMoveForm).from_params(params)
+
+          Admin::MoveQuestions.call(@form, current_user) do
+            on(:ok) do |_question|
+              flash[:notice] = I18n.t("questions_moves.create.success", scope: "decidim.questions.admin")
+              redirect_to EngineRouter.admin_proxy(current_component).root_path
```

Comment:
Maybe we should redirect to the target component? (since we're moving)

Reply:
I thought a bit about it, but it doesn't make much sense - you might want to move resources to different places, or merge them (you're probably "gardening") and that would get you out of the flow.